src/lib/i18n.json

```diff
@@ -165,6 +165,15 @@
     "service-get-display-name-cmd-path-missing-param-err": "Value for path parameter was missing. This is required for running get-display-command. Provide it.",
     "service-get-display-name-cmd-service-name-not-found-err": "Could not find a service for path {0}. Make sure that the specified path is valid.",
 
+    "service-create-cmd-invalid-helm-url-err": "The helm git URL was invalid. Provided helm git URL is an invalid git+ssh or http/https URL: {0}",
+    "service-create-cmd-service-name-missing-err": "Service name was missing. Provide it.",
+    "service-create-cmd-service-path-missing-err": "Service path was missing. Provide it.",
+    "service-create-cmd-service-name-dns-invalid-err": "Service name or DNS value was invalid. See error message for more details.",
+    "service-create-cmd-failed": "Service create command was not successfully executed. Error occurred adding service '{0}' to project '{1}'.",
+    "service-create-cmd-port-invalid-err": "The value for --k8s-service-port was not a valid port number. Provide a valid port number",
+    "service-create-cmd-init-dependency-err": "Could not locate file information in bedrock.yaml. Run `spk project init` before running this command to initialize the project.",
+    "service-create-cmd-init-cvg-dependency-err": "hasVariableGroups was not set file information in bedrock.yaml. Run `spk project init` and `spk project cvg` commands before running this command to initialize the project.",
```

Comment:
Having difficulty understanding this comment `hasVariableGroups was not set file information in bedrock.yaml`

Reply:
@dennisseah

Reply:
can you please help to reword it?